src/omero_zarr/cli.py

```diff
@@ -126,12 +126,6 @@ def _configure(self, parser: Parser) -> None:
             "--output", type=str, default="", help="The output directory"
         )
 
-        parser.add_argument(
-            "--cache_numpy",
```

Comment:
I assume this takes us to 0.6.0 unless we want to tolerate it being set and print a warning message.

Reply:
I think 0.6.0 makes sense, since we also change the default chunking behaviour